plugins/websocket_server/WebsocketServer.cc

```diff
@@ -374,6 +404,9 @@ void WebsocketServer::OnConnect(int _socketId)
 //////////////////////////////////////////////////
 void WebsocketServer::OnDisconnect(int _socketId)
 {
+  if (this->connections.find(_socketId) == this->connections.end())
```

Comment:
I would add the comment `Skip invalid sockets` here as well.

Reply:
Fixed in a30524be7c154097aadcd62f4f840657efabf9f6